data/scripts.js

```diff
@@ -113,6 +113,8 @@ exports.BattleScripts = {
 			pokemon.deductPP(move, extraPP);
 		}
 
+		this.singleEvent('UseMoveMessage', move, null, pokemon, target, move);
```

Comment:
If this is what you're calling the event, then perhaps it should go directly after the `TryMove` instead of before.